collector/lib/NetworkStatusNotifier.cpp

```diff
@@ -326,15 +335,27 @@ void NetworkStatusNotifier::AddConnections(::google::protobuf::RepeatedPtrField<
       // the case where we send a close event for a connection that it doesn't
       // know about.
       //
-      if (!rate_limiter.Allow(delta_entry.first.container())) {
-        CLOG_THROTTLED(INFO, std::chrono::seconds(5)) << "Rate limiting connections reported from container " << delta_entry.first.container();
+      std::string container = delta_entry.first.container();
```

Comment:
I don't think we need to make a copy here, the call to `UnorderedMap::operator[]` will make it on its own if it's needed.
```suggestion
      const std::string& container = delta_entry.first.container();
```

Reply:
Done

Reply:
As discussed offline, gone with a string_view here due to limitations with references as map keys.